### ISSUE TITLE
Add search to list of projects for each user

### ISSUE THREAD

**Issue by nlhkabu** [pypi/warehouse#2819]
On:

- `warehouse/templates/manage/projects.html`
- `warehouse/templates/accounts/profile.html`

Some users have a lot of projects (see https://github.com/pypa/warehouse/issues/2734#issuecomment-358584365) - we need to find a way for these users to find the project they are looking for.

**Comment by pradyunsg** [comment]
I imagine this would be a part of the maintainer MVP; correct?

**Comment by di** [comment]
@pradyunsg I don't think so, since it's just a nicety (ctrl-F still works) and isn't a current feature of pypi-legacy.

**Comment by pradyunsg** [comment]
"Cool but not urgent" then? :P

**Comment by di** [comment]
@pradyunsg Sure.

**Comment by nlhkabu** [comment]
Closing as low priority.
